pkg/controller/clusterautoscaler/clusterautoscaler_controller.go

```diff
@@ -338,8 +338,9 @@ func (r *Reconciler) AutoscalerDeployment(ca *autoscalingv1.ClusterAutoscaler) *
 	}
 
 	annotations := map[string]string{
-		util.CriticalPodAnnotation:    "",
-		util.ReleaseVersionAnnotation: r.config.ReleaseVersion,
+		util.CriticalPodAnnotation:         "",
+		util.ReleaseVersionAnnotation:      r.config.ReleaseVersion,
+		"workload.openshift.io/management": `{"effect": "PreferredDuringScheduling"}`,
```

Comment:
Can we make these strings consts and/or add a link to the enhancement so future us can work out why we have these here?